spec/components/shared/progress_card_component_spec.rb

```diff
@@ -11,12 +11,12 @@
   it 'renders the list of steps' do
     render_inline(described_class.new(submission:))
     expect(page).to have_css('aside h2', text: 'Progress')
-    expect(page).to have_css('li', count: TOTAL_STEPS)
+    expect(page).to have_css('li', count: TOTAL_STEPS + 2)
```

Comment:
Note: I specifically chose not to increase the `TOTAL_STEPS` variables in support as there are various tests that use this value based on the steps that the student interacts with, while this `li` check has 2 extra `li` for the registrar steps that aren't interacted with (same as below).

Reply:
Do you think we should be bumping the `TOTAL_STEPS` soon or is it not at all worth it?  If it's the latter, this may be good to put in as a comment. Alternatively, you could create a sister variable to `TOTAL_STEPS` with a suitable name and its value could be `TOTAL_STEPS + 2`